.github/.htpasswd

```diff
@@ -0,0 +1,2 @@
+# htpasswd -c .htpasswd geth - change password for geth
```

Comment:
im not familiar with this, do we need this file or is this file being used somewhere